net.openhabittracker.OpenHabitTracker.yaml

```diff
@@ -0,0 +1,53 @@
+id: net.openhabittracker.OpenHabitTracker
+#runtime: org.freedesktop.Platform
+#runtime-version: '24.08'
+#sdk: org.freedesktop.Sdk
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet9
+build-options:
+  prepend-path: "/usr/lib/sdk/dotnet9/bin"
+  append-ld-library-path: "/usr/lib/sdk/dotnet9/lib"
+  prepend-pkg-config-path: "/usr/lib/sdk/dotnet9/lib/pkgconfig"
+
+command: OpenHT
+
+finish-args:
+  - --device=dri
+  # TODO: Replace this with wayland and fallback-x11 once Wayland support
+  #       becomes available:
+  #       https://github.com/AvaloniaUI/Avalonia/pull/8003
+  - --socket=x11
+  - --share=ipc
+  - --env=DOTNET_ROOT=/app/lib/dotnet
+
+modules:
+  - name: dotnet
+    buildsystem: simple
+    build-commands:
+    - /usr/lib/sdk/dotnet9/bin/install.sh
+
+  - name: OpenHabitTracker
+    buildsystem: simple
+    sources:
+      - type: git
+        url: https://github.com/Jinjinov/OpenHabitTracker.git
+        tag: 1.1.0.10
+      - ./nuget-sources.json
+    build-options:
+      arch:
+        aarch64:
+          env:
+            RUNTIME: linux-arm64
+        x86_64:
+          env:
+            RUNTIME: linux-x64
+    build-commands:
+      - dotnet publish OpenHabitTracker.Blazor.Photino/OpenHabitTracker.Blazor.Photino.csproj -c Release -f net9.0 -r linux-x64 -p:PublishSingleFile=true -p:SelfContained=true --source ./nuget-sources --source /usr/lib/sdk/dotnet9/nuget/packages
```

Comment:
why still `-r linux-x64`?

Reply:
This is still related to the problematic https://github.com/tryphotino/photino.Blazor NuGet package that prevented the build on the first attempt because it uses `libwebkit2gtk-4.0-37`. Now someone from the community helped me make a custom version of the package that uses `libwebkit2gtk-4.1`, but only for `linux-x64`.